dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/file/LogFileReader.kt

```diff
@@ -68,11 +69,16 @@ internal class LogFileReader(
 
     // region Internal
 
-    private fun deobfuscate(input: ByteArray): String {
-        val output = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            deobfuscateApi26(input)
-        } else {
-            AndroidBase64.decode(input, AndroidBase64.DEFAULT)
+    private fun deobfuscate(input: ByteArray): String? {
+        val output = try {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                deobfuscateApi26(input)
+            } else {
+                AndroidBase64.decode(input, AndroidBase64.DEFAULT)
+            }
+        } catch (e: IllegalArgumentException) {
+            sdkLogger.e("Invalid log found, dropping it", e)
+            return null
```

Comment:
Isn't it worth covering this new code branch with tests? It's quite "core" feature, so maybe we could add it.